src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -389,6 +390,13 @@ public void periodic() {
       setGoToIntake(false);
     } else if (goToIntake && ScoringSubsystem.getInstance().isAlgaeDetected()) {
       setGoToIntake(false);
+    } else if (goToIntake
+        && ModeConstants.currentMode == Mode.REAL
+        && this.getPose()
+                .getTranslation()
+                .getDistance(OTFState.findOTFPoseFromDesiredLocation(this).getTranslation())
+            < 1) {
+      setGoToIntake(false);
```

Comment:
@minhnguyenbhs could we make the "1" a configurable JSON constant. Also, can we add an AND that we need to be in teleop?

If we do these two things we may end up adding this on wednesday. I want to have this as something we can have at our disposal for DCMP if possible since the change is simple

Reply:
@aidnem ^ can you do this? It was a problem in today's driver practice